src/js.rs

```diff
@@ -340,6 +340,14 @@ extern {
     /// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/slice
     #[wasm_bindgen(method, js_class = "String")]
     pub fn slice(this: &JsString, start: u32, end: u32) -> JsString;
+
+    /// The substr() method returns the part of a string between
+    /// the start index and a number of characters after it.
+    ///
+    /// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/substr
+    /// TODO: Add `NaN` and `undefined` support
```

Comment:
Added todo as we don't yet have that support(or at least not that I know of).

Reply:
The `NaN` behavior is on the JS engine implementation side, not our side, so we don't need to worry about it.